app/views/transferConfirmation.scala.html

```diff
@@ -0,0 +1,29 @@
+@()(implicit messages: Messages)
+@main(Messages("transferConfirmation.header")) {
+    @defining(play.core.PlayVersion.current) { version =>
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <a href="@routes.DashboardController.dashboard()" class="govuk-back-link">Back</a>
+                @progressIndicator(Messages("transferConfirmation.progress"))
+                <div class="govuk-panel govuk-panel--confirmation">
+                    <h1 class="govuk-panel__title">
+                        @Messages("transferConfirmation.transferComplete")
+                    </h1>
+                    <div class="govuk-panel__body">
+                        @Messages("transferConfirmation.yourTransferIsComplete")
```

Comment:
There is some additional text on the wireframes before the links to the Dashboard / Logout: https://u6vda4.axshare.com/transfercomplete.html

"_What happens next
We’ve received your transfer and will notify you when your records are in our archive.
Please do not delete any records until then._"

Reply:
Yes, I left that out because this page was just supposed to be a placeholder page, I kept adding stuff (since it was easy) and then before you know, I had done a little too much to it. Should I maybe leave as an indicator to the person that picks it up that it is unfinished/not a lot of time was spent developing it?

Reply:
If its just meant to be a placeholder page and there is a separate JIRA task to create this page, then I'd be tempted to add something to page to make this clear: 
```
...
<span class="govuk-caption-xl">Step 9 of 9</span>
<p>Placeholder page</p>
...
```

Reply:
Ok, I've replaced it